File: Python basics/2.6.03.py
'''

1) Создайте пустой список с возможностью его заполнения с клавиатуры через функцию extend.
2) Выведите в консоль первые 3 символа, начиная со второго символа включчительно.

Sample Input:

крот

Sample Output:

['р', 'о', 'т']

'''

a = []
a.extend(input())
print(a[1:4:])

File: JavaScript for Beginners/2  Стандартные объекты/2.6.03.js
/*

В этом задании в нашу функцию передаются 4 числа. Вам необходимо вычислить результат деления большего из этих чисел на меньшее, 
и округлив до ближайшего меньшего целого вернуть из функции.

Sample Input:

1 4 8 8

Sample Output:

8

*/

function testMath(a, b, c, d) {
    let min_n = Math.min(a,b,c,d);
    let max_n = Math.max(a,b,c,d);
    let res = Math.floor(max_n/min_n);
    return res;
}
